s3/README.md

```diff
@@ -4,44 +4,75 @@ Module to create s3 bucket
 
 ## Input Variables
 
-- region
```

Comment:
Add the features and limitations of S3 bucket

Reply:
Added